gmso/tests/test_lammps.py

```diff
@@ -1,14 +1,53 @@
 from gmso.core.box import Box
-from gmso.formats.lammpsdata import write_lammpsdata
+from gmso.formats.lammpsdata import write_lammpsdata, read_lammpsdata
 from gmso.tests.base_test import BaseTest
+from gmso.tests.utils import get_path
+import unyt as u
 
 
 class TestLammpsWriter(BaseTest):
     def test_write_lammps(self, topology_site):
-        top = topology_site()
-        write_lammpsdata(top, filename='data.lammps')
+        write_lammpsdata(topology_site(), filename='data.lammps')
 
     def test_write_lammps_triclinic(self, topology_site):
-        top = topology_site()
-        top.box = Box(lengths=[1,1,1], angles=[60,90,120])
-        write_lammpsdata(top, filename='data.triclinic')
+        topology_site().box = Box(lengths=[1,1,1], angles=[60,90,120])
```

Comment:
Not sure if this will give an error or not, but I am not sure if those angles and lengths will work. It might not matter, but I am not sure